unit-tests/src/test/java/org/eclipse/collections/impl/bag/mutable/SynchronizedBagTest.java

```diff
@@ -302,4 +303,11 @@ public void selectUnique()
         MutableSet<String> actual = bag.selectUnique();
         Assert.assertEquals(expected, actual);
     }
+
+    @Test
+    public void distinctViewSynchronized()
+    {
+        RichIterable<Integer> actual = this.newWith(1, 1, 2, 3, 4, 4).distinctView();
+        Verify.assertInstanceOf(SynchronizedRichIterable.class, actual);
```

Comment:
Please add an assertion that the distinct elements are `1, 2, 3, 4` as well.

Reply:
I added more tests in multiple places and this class should be covered as well :)